import_export/forms.py

```diff
@@ -1,29 +1,48 @@
 import os.path
 
 from django import forms
+from django.conf import settings
 from django.contrib.admin.helpers import ActionForm
 from django.utils.translation import gettext_lazy as _
 
 
 class ImportForm(forms.Form):
     import_file = forms.FileField(
         label=_('File to import')
-        )
+    )
     input_format = forms.ChoiceField(
         label=_('Format'),
         choices=(),
-        )
+    )
 
     def __init__(self, import_formats, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        choices = []
-        for i, f in enumerate(import_formats):
-            choices.append((str(i), f().get_title(),))
+        choices = [
+            (str(i), f().get_title())
+            for i, f in enumerate(import_formats)
+        ]
         if len(import_formats) > 1:
             choices.insert(0, ('', '---'))
+            self.fields['import_file'].widget.attrs['class'] = 'guess_format'
+            self.fields['input_format'].widget.attrs['class'] = 'guess_format'
 
         self.fields['input_format'].choices = choices
 
+    class Media:
+        if settings.DEBUG:
+            js = (
+                'admin/js/vendor/jquery/jquery.js',
+                'admin/js/jquery.init.js',
+                'import_export/guess_format.js',
+            )
+        else:
+            js = (
+                'admin/js/vendor/jquery/jquery.min.js',
+                'admin/js/jquery.init.js',
+                'import_export/guess_format.js',
+            )
```

Comment:
It would be ideal to avoid duplicating the `jquery.init.js` and `guess_format.js` declarations in this block.  Do you agree, or is there a reason we cannot do this?

Reply:
@KommuSoft I'd like to get this PR into the next release.  Please could you provide a response to this comment?  thanks

Reply:
Hi, I used approximately the same solution as in https://github.com/django/django/blob/0dd29209091280ccf34e07c9468746c396b7778e/django/contrib/admin/options.py#L729-L732

Reply:
No problem with the approach you have taken.  I asking whether we could remove the duplication (i.e. `init.js` and `guess_format.js` declared twice).
thanks

Reply:
Ignore previous comment - looks like you resolved this - thanks